dataclient/dataclient.go

```diff
@@ -0,0 +1,60 @@
+package dataclient
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/zalando-incubator/skrop/filters"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/eskipfile"
+	"github.com/zalando/skipper/routing"
+)
+
+type skropDataClient struct {
+	fileName string
+	prepend  *eskip.Filter
+	append   *eskip.Filter
+}
+
+func NewSkropDataClient(eskipFile string) routing.DataClient {
+
+	emptyArgs := make([]interface{}, 0)
+
+	pre := &eskip.Filter{
+		Name: filters.NewFinalizeResponse().Name(),
+		Args: emptyArgs,
+	}
+	post := &eskip.Filter{
+		Name: filters.NewSetupResponse().Name(),
+		Args: emptyArgs,
+	}
+	return skropDataClient{
+		fileName: eskipFile,
+		prepend:  pre,
+		append:   post,
+	}
+}
+
+func (s skropDataClient) LoadAll() ([]*eskip.Route, error) {
+	f, err := eskipfile.Open(s.fileName)
+	if err != nil {
+		log.Error("Error while opening eskip file", err)
```

Comment:
minor: it is recommended not to capitalize error messages so that they can be composed nicer